internal/infrastructure/storage/db/pg/migration/20230504102232_init.up.sql

```diff
@@ -0,0 +1,63 @@
+CREATE TABLE market (
+    name VARCHAR(50) NOT NULL PRIMARY KEY,
+    base_asset VARCHAR(255) NOT NULL,
+    quote_asset VARCHAR(255) NOT NULL,
+    base_asset_precision INTEGER NOT NULL DEFAULT 8,
+    quote_asset_precision INTEGER NOT NULL DEFAULT 8,
```

Comment:
Remove defaults here, they are handled at domain level - actually, the default value used at domain level is 0.
```suggestion
    base_asset_precision INTEGER NOT NULL,
    quote_asset_precision INTEGER NOT NULL,
```